frb_codegen/src/config/opts.rs

```diff
@@ -27,6 +28,7 @@ pub struct Opts {
     pub skip_deps_check: bool,
     pub wasm_enabled: bool,
     pub inline_rust: bool,
+    pub no_use_bridge_in_method: bool,
```

Comment:
uber-nit: shall we name it `use_bridge_in_method` here, since it is usually to avoid such negations. The `no_...` in `RawOpts` is acceptable since we do want that flag for end users.

Reply:
I am follow code `no_build_runner: config.no_build_runner`.
Although my point of view is that it's not a good idea to change internally